assets/js/hooks/Mapper/components/map/styles/index.scss

```
@@ -1,2 +1,5 @@
-@use './default-theme.scss'; 
-@use './pathfinder-theme.scss'; 
\ No newline at end of file
+@use './default-theme.scss';
```

Stylelint (17.14.1)

[error] Unexpected extension ".scss" in `@use` (scss/load-partial-extension)

(scss/load-partial-extension)